components/carousel/utils/useCarouselEvents.ts

```diff
@@ -151,39 +154,41 @@ const limitDisplayNfts = (data) => {
 
 export const useCarouselNftEvents = ({ type }: Types) => {
   const nfts = ref<CarouselNFT[]>([])
-  const items = computed(() => limitDisplayNfts(nfts.value))
-  const chains = ['ahk', 'ahp', 'rmrk', 'ksm']
-
-  onMounted(async () => {
-    for (const chain of chains) {
-      useChainEvents(chain, type, limit, null).then(({ data }) =>
-        nfts.value.push(...flattenNFT(data.value, chain)),
-      )
-    }
+  const items = computed(() => sortNfts(nfts.value))
+
+  // total 30 items will show up on the carousel
+  // sum from limits on the third parameter of useEvents
+  const { data: ahp } = useEvents('ahp', type, 15)
+  const { data: ahk } = useEvents('ahk', type, 9)
+  const { data: ksm } = useEvents('ksm', type, 3)
+  const { data: rmrk } = useEvents('rmrk', type, 3)
+
```

Comment:
Could we make a config like this? `[ { chain: 'ahp', limit: 15}]`

Reply:
updated, how about now

Reply:
looks great